### DISCUSSION TITLE
Pacman Package dont have last updates

### DISCUSSION THREAD

**Discussion by GaNiziolek** [dbeaver/dbeaver#22083] [Q&A] [answered]
### Description

The last version on Arch official packages was 23.2.2, this version has many bugs that have already been fixed but I can't upgrade because this package is not updated on Arch.

https://archlinux.org/packages/extra/x86_64/dbeaver/

### DBeaver Version

Community Edition 23.2.2

### Operating System

Arch Linux

**Comment by Destrolaric** [comment, marked as answer]
@GaNiziolek Hi. Unfortunately, the Pacman package is community-maintained. DBeaver team is not updating the package, and we suggest contacting the package maintainer regarding package updates. 
You can also use our flatpak image, which our team maintains.

**Comment by GaNiziolek** [comment]
I've made the PKGBUILD to the last version here https://github.com/muflone/pkgbuilds/pull/26, I think that @muflone will soon made the update on Arch Packages

**Comment by muflone** [comment]
dbeaver 23.3.0-1 was just pushed in the `extra` repository
sorry for the delay
